pkg/restapi/operation/didexchange/didexchange_test.go

```diff
@@ -389,3 +392,36 @@ func TestOperationEventError(t *testing.T) {
 	err = client.UnregisterActionEvent(aCh)
 	require.NoError(t, err)
 }
+
+func TestHandleMessageEvent(t *testing.T) {
+	op, err := New(&mockprovider.Provider{StorageProviderValue: mockstore.NewMockStoreProvider(),
+		ServiceValue: &protocol.MockDIDExchangeSvc{}}, webhook.NewHTTPNotifier(nil))
+	require.NoError(t, err)
+	require.NotNil(t, op)
+
+	op.handleMessageEvents(service.StateMsg{Type: service.PostState, Properties: "invalid didex prop type"})
+	op.handleMessageEvents(service.StateMsg{Type: service.PostState, Properties: errors.New("err type")})
```

Comment:
This test doesn't check for anything beyond this point, so it looks like it's always going to pass.

Reply:
https://github.com/hyperledger/aries-framework-go/pull/580#discussion_r338776651